server/src/main/java/com/google/musicanalysis/site/OAuthCallbackServlet.java

```diff
@@ -85,5 +93,9 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
 
     res.setContentType("text/html");
     res.getWriter().printf("<h1>the access token for %s is %s</h1>", getServiceName(), accessToken);
+    
+    // store access token in a session as youtube_access_token or spotify_access_token
+    HttpSession session = req.getSession();
+    session.setAttribute(getSessionTokenKey(), accessToken.toString());
```

Comment:
2 things:
1, this should be `this.getSessionTokenKey()`

2, Is there a reference to `getSessionTokenKey()` in the Spotify OAuth Servlet? If not, this will likely introduce errors or unintended side effects. You should update that file as well.

Reply:
Fixed `getSessionTokenKey()`. `this.getSessionTokenKey()` and `getSessionTokenKey()` both work? Other functions on the top of this `OauthCallbackServlet.java` file are also called without calling `this.` such as `getRedirectUri()`, `getClientId()`, ... `getServiceName()` within this file.